AmplifyPlugins/Geo/Sources/AWSLocationGeoPlugin/Dependency/AWSDeviceTracker.swift

```diff
@@ -62,14 +57,20 @@ class AWSDeviceTracker: NSObject, CLLocationManagerDelegate, AWSDeviceTrackingBe
     func startTracking(for device: Geo.Device) throws {
         networkMonitor.start()
         UserDefaults.standard.removeObject(forKey: AWSDeviceTracker.deviceIDKey)
```

Comment:
should this objects get removed on user logout as well? should device tracking stop on user logout?